packages/garbo/src/diet.ts

```diff
@@ -276,6 +276,16 @@ export function nonOrganAdventures(): void {
   if (globalOptions.ascend) {
     useIfUnused($item`borrowed time`, "_borrowedTimeUsed", 20 * MPA);
   }
+
+  while (get("_extraTimeUsed", 3) < 3) {
```

Comment:
tbh while loops like this always sketch me out, because if mafia somehow fails to update the property it's gonna infinite loop. can you instead calculate how many extra time should be used without a loop then just use that many?